src/palindrom.js

```diff
@@ -341,15 +363,17 @@ const Palindrom = (() => {
           return;
         }
 
-        const m = {
-          statusText: 'WebSocket connection could not be made.',
-          readyState: this._ws.readyState,
-          url: upgradeURL
-        };
+        const message = [
+          'WebSocket connection could not be made',
+          'readyState: ' + this._ws.readyState
+        ].join('\n');
```

Comment:
You can use:

Reply:
Yes, do you thing it would look good without indentation? I don't mind either ways. But I lean to good looks here because perf is irrelevant, since the app is dead at this point.